gp-includes/routes/profile.php

```diff
@@ -16,17 +16,22 @@ public function profile_get() {
 		$this->tmpl( 'profile' );
 	}
 
-	public function profile_post() {
+	public function profile_post( $user_id = null ) {
 		if ( isset( $_POST['submit'] ) ) {
+			if ( (int)$user_id == 0 ) { 
```

Comment:
It would be nice to know where the 0 comes from. IMO is this a case where a simple `if ( ! $user_id )` would be enough.

Related: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions
=> `if ( 0 == (int) $user_id ) { ` or better:

```
$user_id = (int) $user_id;
if ( 0 === $user_id ) { 
…
}
```